contrib/epee/src/file_io_utils.cpp

```diff
@@ -102,29 +102,6 @@ namespace file_io_utils
 	}
 
 
-	bool get_file_time(const std::string& path_to_file, time_t& ft)
-	{
-		boost::system::error_code ec;
-		ft = boost::filesystem::last_write_time(boost::filesystem::path(path_to_file), ec);
-		if(!ec)
-			return true;
-		else
-			return false;
-	}
-
-
-	bool set_file_time(const std::string& path_to_file, const time_t& ft)
```

Comment:
I think it could be useful and nicely wraps boost::filesystem's heavy headers.

Reply:
On the other hand, `boost::filesystem` functions are already mentioned 245 times in 9 files throughout the project, and I believe boost already has functions for `get_file_time` and `set_file_time`: [https://www.boost.org/doc/libs/1_58_0/libs/filesystem/doc/reference.html#last_write_time](https://www.boost.org/doc/libs/1_58_0/libs/filesystem/doc/reference.html#last_write_time)

Reply:
You have to start from somewhere and lead by a good example, even if at the beginning it appears to make no sense, because it's already much worse somewhere else. Bit by bit the better solutions can purge the worse :)